package.json

```diff
@@ -39,8 +38,7 @@
     "ember-export-application-global": "^1.0.5",
     "ember-load-initializers": "^0.6.0",
     "ember-resolver": "^2.0.3",
-    "ember-source": "^2.12.0-beta.1",
-    "ember-welcome-page": "^2.0.2",
+    "ember-source": "^2.11.0",
```

Comment:
Do you want to lock this <= ~2.11.0 so in development you don't start mistakenly use the native implementation.  (it trolled me for a few minutes).